### ISSUE TITLE
ossia sends websocket OSC messages to clients that haven't subscribed to them

### ISSUE THREAD

**Issue by x37v** [ossia/libossia#663]
On the latest master. If my ossia application response to a value update by creating new nodes, those values are sent out via the websocket even to clients that haven't sent a `LISTEN` command.

The https://github.com/Cycling74/libossia/tree/xnor/bug-example branch I've created as an `examples/bug` project that illustraits another bug, but to my understanding of the OSCQuery proposal, this example shouldn't actually get *any* websocket OSC messages sent to the javascript client because the client hasn't subscribed via `LISTEN`.

**Comment by jcelerier** [comment]
indeed, I think that we had discussed that at some point with @bltzr ?
should not be hard to fix, I wonder if it would break some workflow of someone somewhere though

**Comment by x37v** [comment]
> indeed, I think that we had discussed that at some point with @bltzr ?
> should not be hard to fix, I wonder if it would break some workflow of someone somewhere though

I figure it would break someone's workflow but is actually more "correct" RE the proposal document. Could always add a compile-time and/or runtime switch to enable the old behavior.
